pyproject.toml

```diff
@@ -1,24 +1,20 @@
 [tool.poetry]
 name = "griptape-chat"
-version = "0.2.0"
+version = "0.29"
```

Comment:
We don't need/want this to be tightly versioned with griptape. Maybe we change it to `1.0.0` now that is is fully functional